shared/macros/bash.jinja

```diff
@@ -1688,14 +1688,17 @@ fi
     Remediation for grub2 bootloader arguments
 #}}
 {{% macro grub2_bootloader_argument_remediation(ARG_NAME, ARG_NAME_VALUE) %}}
+{{% set grub_helper_args = ["--update-kernel=ALL", "--args= {{{ ARG_NAME_VALUE }}}"] -%}}
```

Comment:
The Shellcheck finding is legit;
The files he complains about is as:
```bash
var_slub_debug_options='(bash-populate var_slub_debug_options)'
grubby --update-kernel=ALL --args=
```
while in master it is:
```bash
var_slub_debug_options='(bash-populate var_slub_debug_options)'
grubby --update-kernel=ALL --args=slub_debug=$var_slub_debug_options 
```

The problem is on this line, which changed from 
`{{% set grub_helper_args = ["--update-kernel=ALL", "--args=" ~ ARG_NAME_VALUE] -%}}`
to 
`{{% set grub_helper_args = ["--update-kernel=ALL", "--args= {{{ ARG_NAME_VALUE }}}"] -%}}`

Reply:
So, change line back to:
{{% set grub_helper_args = ["--update-kernel=ALL", "--args=" ~ ARG_NAME_VALUE] -%}}
Correct?

Reply:
I think so, :)